source/connect/stable-api.txt

```diff
@@ -21,4 +21,110 @@ Stable API
 
    The Stable API feature requires {+mdb-server+} 5.0 or later.
 
-.. TODO
\ No newline at end of file
+Overview
+--------
+
+In this guide, you can learn how to specify **{+stable-api+}** compatibility when 
+connecting to a MongoDB deployment.
+
+The {+stable-api+} feature forces the server to run operations with behaviors compatible 
+with the API version you specify. Using the {+stable-api+} ensures consistent responses 
+from the server and provides long-term API stability for your application.
+
+The following sections describe how you can enable and customize {+stable-api+} for
+your MongoDB client. For more information about the {+stable-api+}, including a list of 
+the commands it supports, see :manual:`Stable API </reference/stable-api/>` in the
+{+mdb-server+} manual.
+
+Enable the {+stable-api+}
+-------------------------
+
+To enable the {+stable-api+}, perform the following steps:
+
+1. Construct a ``ServerApi`` object and specify a {+stable-api+} version. You must use
+   a {+stable-api+} version defined in the ``ServerApiVersion`` enum.
+#. Construct a ``MongoClientSettings`` object using the ``MongoClientSettings.Builder`` class.
+#. Instantiate a ``MongoClient`` using the ``MongoClient.create()`` method and 
```

Comment:
S: the code example doesn't use the create() method, so I'd just say the constructor
```suggestion
#. Instantiate a ``MongoClient`` using the constructor and 
```